CMakeLists.txt

```diff
@@ -209,6 +181,20 @@ if (NOT OpenCV_FOUND)
 endif ()
 set(LBANN_HAS_OPENCV ${OpenCV_FOUND})
 
+# CUDA-ness of LBANN is 1:1 with Hydrogen. Iff Hydrogen has CUDA, LBANN gets CUDA.
```

Comment:
Do you really mean if-and-only-if hydrogen has CUDA, or is the iff just a typo.

Reply:
I mean if and only if. #mathematicianed.